src/content/configuration/entry-context.mdx

```diff
@@ -120,7 +121,7 @@ module.exports = {
 };
 ```
 
-The `app` chunk will not contain the modules that `react-vendors` have.
```

Comment:
I think not is correct here, as we have defined `dependOn: 'react-vendors'` in the config above.

Reply:
Oh, I see. Yes, I think understand the wording better now